mod/metrics/set/type/metric/tabs.rb

```diff
@@ -19,7 +19,7 @@ def tab_list
   end
 
   def nest_about
-    return "" if card.about_card.new?
+    # return "" if card.about_card.new?
```

Comment:
Laureen specifically requested the "add about" functionality.  not sure why we removed it...